### DISCUSSION TITLE
Close Samples

### DISCUSSION THREAD

**Discussion by duncanca** [jodyphelan/TBProfiler#352] [General]
I have upgraded from  v4.4.2 to 6.2.0.  Has the close sample option been removed?  I am using the ---call_whole_genome  and --snp_dist arguments, no errors are generated but the close samples report section is gone.  A binary file snp_diffs.db is generated but I am unsure how to visualize this data.

I found the close sample report very useful to quickly identify related cases and I am hoping this will still be supported.

**Comment by jodyphelan** [comment]
Hi @duncanca ,

In the json files they are available under the key "linked_samples". However I noticed now that this information is not present in the text output so I'll aim to put that back in for the next release

**Comment by duncanca** [comment]
I see the data in the json file. Thanks for all your work in developing and
continuing to improving this tool.
